README.md

```diff
@@ -7,3 +7,38 @@
 1. Аля Новикова
 2. Ваня Павлов
 3. Ваня Олейник
+
+
+## Запуск 
+
+Скачайте докер и запустите команду:
+```
+docker-compose up
+```
+
+## Пример запросов:
+
+Регистрация
+```
+requests.post("http://localhost:8080/signup", json={"login": "ivanpavlov", "password": "SomeComplicated2131"})
```

Comment:
What is this? What is Python code doing here?

Reply:
That is my way of doing post/get requests. (I can't use curl on windows properly). You might think this is pseudocode.  Seems readable to me.

Reply:
This is all very well, but please think about the readers who might be misled by this. You'd better mention the language of choice explicitly, like "from Python .. run this" or "using curl .. run that".

Reply:
And of course you know how to spell "usage"  :)